### PR TITLE
Add live data to the homepage

### PR DESCRIPTION
**Pull request by cdedreuille**
This PR adds live data for the homepage:
- NPM downloads
- Contributor count
- Discord members
- X followers (WIP)
- Youtube followers (WIP)

### CODE REVIEW

File: apps/frontpage/app/page.tsx [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
 export default async function Page() {
-  const githubCount = await getGithubCount();
+  const githubCount = await fetchGithubCount();
+  const { formattedResult: npmDownloads } = await fetchNpmDownloads();
+  const { formattedResult: contributorCount } =
+    await fetchGithubContributorCount();
+  const { formattedResult: discordMembers } = await fetchDiscordMembers();
 
-  return <Home githubCount={githubCount} />;
+  console.log(discordMembers);
```
Comment (kylegach) [inline apps/frontpage/app/page.tsx:16]: Remove?

File: libs/utils/src/lib/fetch-github-count.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export const getGithubCount = async () => {
+export const fetchGithubCount = async () => {
```
Comment (kylegach) [inline libs/utils/src/lib/fetch-github-count.ts:1]: This one should return the same `{ number, formattedResult }` shape as the others.

File: apps/frontpage/app/page.tsx (hunk 1 above)
Comment (cdedreuille) [reply apps/frontpage/app/page.tsx]: Removed.

File: libs/utils/src/lib/fetch-github-count.ts (hunk 2 above)
Comment (cdedreuille) [reply libs/utils/src/lib/fetch-github-count.ts:1]: You're right. I corrected that.

File: apps/frontpage/components/home/social-validation/social-validation.tsx [hunk 3]
```diff
@@ -334,7 +336,7 @@ export const SocialValidation: FC = () => {
             <a href="https://twitter.com/storybookjs">Follow on Twitter</a>
           </Button>
           <div>
-            <div className="text-sm text-white">{`${twitterFollowerCount.toLocaleString()}+`}</div>
+            <div className="text-sm text-white">24,200+</div>
```
Comment (kylegach) [inline apps/frontpage/components/home/social-validation/social-validation.tsx:339]: I'd rather keep these hardcoded values as constants, to aid semi-regular updates.

```suggestion
            <div className="text-sm text-white">{`${TWITTER_FOLLOWER_COUNT}+`}</div>
```

Comment (kylegach) [reply apps/frontpage/components/home/social-validation/social-validation.tsx:339]: Alternatively, you could go ahead and implement `getTwitterFollowerCount` and just have it return a constant. Either approach is fine, imo.

**Review by kylegach — APPROVED** [review]
👍

File: apps/frontpage/components/home/social-validation/social-validation.tsx (hunk 3 above)
Comment (cdedreuille) [reply apps/frontpage/components/home/social-validation/social-validation.tsx:339]: I don't think this make sense to make them a constant if this is to be used only in one place. You make them a constant if you want to use them in multiple places.

**Pull request merged**